src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { NewTodoPropTypes } from '../../constants/proptypes';
+
+class NewTodo extends React.Component {
+  state = {
+    inputValue: '',
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    const { inputValue } = this.state;
+    const { onAdd } = this.props;
+
+    if (inputValue) {
+      onAdd(inputValue);
+
+      this.setState({
+        inputValue: '',
+      });
+    }
+  };
+
+  handleInputChange = (event) => {
```

Comment:
вариант деструктуризации:
```suggestion
  handleInputChange = ({ target: { value } }) => {
```